aspnetcore/host-and-deploy/blazor/webassembly.md

```diff
@@ -325,3 +325,59 @@ The `--urls` argument sets the IP addresses or host addresses with ports and pro
 ## Configure the Linker
 
 Blazor performs Intermediate Language (IL) linking on each Release build to remove unnecessary IL from the output assemblies. For more information, see <xref:host-and-deploy/blazor/configure-linker>.
+
+## Custom boot resource loading
+
+A Blazor WebAssembly app can use `loadBootResource` to override the built-in boot resource loading mechanism. Use `loadBootResource` for the following scenarios:
+
+* Allow users to load static resources, such as timezone data or *dotnet.wasm* from a CDN.
+* Load compressed assemblies using an HTTP request and decompress them on the client for hosts that don't support fetching compressed contents from the server.
+* Alias resources to a different name by redirecting each `fetch` request to a new name.
+
+`loadBootResource` parameters appear in the following table.
+
+| Parameter    | Description |
+| ------------ | ----------- |
+| `type`       | The type of the resource. Permissable types: `assembly`, `pdb`, `dotnetjs`, `dotnetwasm`, `timezonedata` |
+| `name`       | The name of the resource. |
+| `defaultUri` | The relative or absolute URI of the resource. |
+| `integrity`  | The integrity string representing the expected content in the response. |
+
+`loadBootResource` returns any of the following to override the loading process:
+
+* URI string. Example: `https://my-awesome-cdn.com/blazorwebassembly/3.2.0/${name}`
+* `Promise<Response>`. Example: `return fetch(someUrl);`
+* `null`/`undefined`, which results in the default loading behavior.
+
+External sources must return the required CORS headers for browsers to allow the cross-origin resource loading. CDNs usually provide the required headers by default.
+
```

Comment:
I'd also call out that if users are issuing their own fetch requests for resources in the `loadBootJson` then they should pass the integrity property in the header.

Reply:
`loadBootJson` isn't described here (or anywhere in the docs), so it needs more explanation.

Reply:
@guardrex I think @captainsafia was still referring to `loadBootResource`. If you make your fetch requests in `loadBootJson` then you need to pass the integrity value properly. See the example I added to the RC announcement post for this: https://devblogs.microsoft.com/aspnet/blazor-webassembly-3-2-0-release-candidate-now-available/

Reply:
I see ... I'll address that now here.